quickstart/docker/image/run-router.sh

```diff
@@ -53,21 +54,20 @@ if [ ! -f "${_UNIQUE_NAME}" ]; then
     echo "CREATING PRIVATE ROUTER CONFIG: ${ZITI_ROUTER_NAME}"
     createPrivateRouterConfig "${ZITI_ROUTER_NAME}"
   fi
-
-  found=$("${ZITI_BIN_DIR-}/ziti" edge list edge-routers 'name = "'"${ZITI_ROUTER_NAME}"'"' | grep -c "${ZITI_ROUTER_NAME}")
-  if [[ found -gt 0 ]]; then
-    echo "----------  Found existing edge-router ${ZITI_ROUTER_NAME}...."
   else
-    "${ZITI_BIN_DIR}/ziti" edge create edge-router "${ZITI_ROUTER_NAME}" -o "${ZITI_HOME}/${ZITI_ROUTER_NAME}.jwt" -t -a "${ZITI_ROUTER_ROLES}"
-    sleep 1
-    echo "---------- Enrolling edge-router ${ZITI_ROUTER_NAME}...."
-    "${ZITI_BIN_DIR}/ziti" router enroll "${ZITI_HOME}/${ZITI_ROUTER_NAME}.yaml" --jwt "${ZITI_HOME}/${ZITI_ROUTER_NAME}.jwt"
-    echo ""
-  fi
-  echo "system initialized. writing marker file"
-  echo "system initialized" > "${_UNIQUE_NAME}"
+    echo " Found existing config file ${_CONFIG_PATH}, not creating a new config."
+    echo " If you would like to overwrite this config, delete the file and restart the container."
+fi
+
+found=$("${ZITI_BIN_DIR-}/ziti" edge list edge-routers 'name = "'"${ZITI_ROUTER_NAME}"'"' | grep -c "${ZITI_ROUTER_NAME}")
+if [[ found -gt 0 ]]; then
+  echo "----------  Found existing edge-router ${ZITI_ROUTER_NAME}...."
 else
-  echo "system has been initialized. starting the process."
+  "${ZITI_BIN_DIR}/ziti" edge create edge-router "${ZITI_ROUTER_NAME}" -o "${ZITI_HOME}/${ZITI_ROUTER_NAME}.jwt" -t -a "${ZITI_ROUTER_ROLES}"
```

Comment:
this should be done in the "if the config doesn't exist" block ?

Reply:
See previous comment, I can put them back in one `if`, just figured this solution was more flexible.

Reply:
I'd rather them delete the container/file if they want to recreate the router so yah, i think one if is best